pkg/distributor/distributor.go

```diff
@@ -769,16 +771,18 @@ func (d *Distributor) trackDiscardedData(
 	req *logproto.PushRequest,
 	validationContext validationContext,
 	tenantID string,
-	validatedLineCount int,
-	validatedLineSize int,
+	validationMetrics validationMetrics,
 	reason string,
+	tenantRetentionHours string,
```

Comment:
May this be part of `validationMetrics` alike the stream retention hours?

Reply:
done , wdyt?